model_signing/serialize.py

```diff
@@ -308,6 +310,15 @@ def serialize_v1(path: Path, chunk: int, signature_path: Path,
         # Finally, we hash everything.
         return hashlib.sha256(bytes(all_hashes)).digest()
 
+    def serialize_v1(path: Path, chunk: int, signature_path: Path,
```

Comment:
note: to simplify tests, I created `_serialize_v1` which takes as input a shard size